packages/block-library/src/group/block.json

```diff
@@ -13,6 +13,10 @@
 		},
 		"customTextColor": {
 			"type": "string"
+		},
+		"tagName": {
```

Comment:
There should probably be some way of limiting this attribute to certain whitelisted values (div, section, header, footer, and aside are what I'm thinking).

Reply:
I'm thinking it could be less of a hard limit and more of a recommendation while in the editor, similarly to how the editor advises against incorrect heading levels or against low-contrast colour combinations.

Reply:
Yes, I imagine it could also have support as a wrapper for custom-elements.